runtime/pendulum/Cargo.toml

```diff
@@ -224,6 +225,51 @@ runtime-benchmarks = [
 ]
 
 try-runtime = [
-  "frame-executive/try-runtime",
-  "frame-try-runtime/try-runtime",
+    "frame-executive/try-runtime",
+    "frame-try-runtime",
+    "frame-system/try-runtime",
+    "parachain-info/try-runtime",
+    "pallet-timestamp/try-runtime",
+    "pallet-aura/try-runtime",
+    "pallet-authorship/try-runtime",
+    "pallet-balances/try-runtime",
+    "pallet-bounties/try-runtime",
+    "pallet-child-bounties/try-runtime",
+    "pallet-collective/try-runtime",
+    "pallet-contracts/try-runtime",
+    "pallet-democracy/try-runtime",
+    "pallet-identity/try-runtime",
+	  "pallet-multisig/try-runtime",
+    "pallet-preimage/try-runtime",
+	  "pallet-proxy/try-runtime",
```

Comment:
The indentation here and in the other Cargo.toml file is a bit off

Reply:
Strange that cargo format doesn't fix this automatically.

Reply:
Also strange that locally I cannot see the wrong indentation!

Reply:
I think Cargo.toml only checks the .rs files and no .toml files unfortunately 🤔